File-Formats/DocIO/Working-with-Word-document.md

```diff
@@ -2216,6 +2216,186 @@ Please download the helper files from the following link to save the stream as a
 
 {% endtabs %} 
 
+### Link Paragraph and Character Style
+
+You can link character styles with paragraph and vice versa in a Word document using LinkedStyleName property.
+
+The following code example explains how to link character and paragraph style.
+
+{% tabs %} 
+
+{% highlight c# %}
+//Creates a Word document
+WordDocument document = new WordDocument();
```

Comment:
Modify the code with using statement in all platforms

Reply:
modified with using statement